definitions/pipelines/somatic_exome.cwl

```diff
@@ -562,6 +577,7 @@ steps:
             normal_sample_name: normal_sample_name
             vep_custom_annotations: vep_custom_annotations
             known_variants: known_variants
+            
```

Comment:
I wouldn't separate the `out` from the rest of the step definition unless we're going to pursue some large-scale formatting changes.